Bio/SeqFeature.py

```diff
@@ -925,6 +925,25 @@ def __radd__(self, other):
             return self._shift(other)
         else:
             return NotImplemented
+    
+    def __sub__(self,other):
+        """Subtracting an integer will give a new FeatureLocation with
+        its start and end offset shifting back by that amount. For example:
+            
+        >>> print(f1)
```

Comment:
The example needs to be self contained, so should start by defining the feature.

Reply:
The ``from Bio.SeqFeature import FeatureLocation`` line needs the ``>>> `` (or can be left out).